.github/workflows/coding-standards.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         dependencies:
           - "locked"
         php-version:
-          - "7.4"
```

Comment:
We should probably run CS checks on 7.4, unless there's a reason to run them on newer versions (some CS checks use the PHP version in the runtime, sometimes)

Reply:
There was a reason I had to bump back then, but maybe this has been resolved in the meantime. I'll have a look.

Reply:
Reverted